manage-data/data-store/templates.md

```diff
@@ -8,27 +8,80 @@ products:
   - id: elasticsearch
 ---
 
-# Templates [index-templates]
+# Templates [elasticsearch-templates]
 
-::::{note}
-This topic describes the composable index templates introduced in {{es}} 7.8. For information about how index templates worked previously, see the [legacy template documentation](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-template).
-::::
+Templates are tools used to construct indices in {{es}}. They specify the properties that each created index will have.
 
+You configure templates prior to creating indices. This way, when each index is created, either manually or through the indexing of a document, the template settings are used as a basis for creating the index. When used with a [data stream](/manage-data/data-store/data-streams.md), a template configures the stream's backing indices as they are created.
 
-$$$getting$$$
-An index template is a way to tell {{es}} how to configure an index when it is created. For data streams, the index template configures the stream’s backing indices as they are created. Templates are configured **prior to index creation**. When an index is created - either manually or through indexing a document - the template settings are used as a basis for creating the index.
+There are two types of template:
 
-There are two types of templates: index templates and [component templates](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-component-template). Component templates are reusable building blocks that configure mappings, settings, and aliases. While you can use component templates to construct index templates, they aren’t directly applied to a set of indices. Index templates can contain a collection of component templates, as well as directly specify settings, mappings, and aliases.
+* An [**index template**](#index-templates) is used to configure indices when they are created, by specifying the mappings, settings, and aliases that each index inherits.
 
-The following conditions apply to index templates:
+* A [**component template**](#component-templates) is a reusable building block used to construct index templates.
 
-* Composable templates take precedence over legacy templates. If no composable template matches a given index, a legacy template may still match and be applied.
+The following conditions apply to using templates:
+
+* Composable index templates take precedence over legacy templates. If no composable template matches a given index, a legacy template may still match and be applied.
```

Comment:
Can we add a link in `legacy templates`? The concept of legacy templates doesn't appear in the doc, and might be difficult for a reader to understand what's about.

Reply:
Thanks for providing that link. I changed this line to:

> Composable index templates take precedence over any [legacy templates](https://www.elastic.co/guide/en/elasticsearch/reference/8.18/indices-templates-v1.html), which were deprecated in {{es}} 7.8.